code/modules/projectiles/updated_projectiles/guns/flamer.dm

```diff
@@ -142,30 +144,27 @@
 	var/burnlevel
 	var/burntime
 	var/fire_color = "red"
-	switch(current_mag.caliber)
-		if("UT-Napthal Fuel") //This isn't actually Napalm actually
```

Comment:
unrelated changes

Reply:
The switching of the flamer ammo types was bad so I slightly reworked it into something better.